canonical/lxd-ui
# Show change counts on save buttons for edit forms

## Done

- Show change counts on save buttons for edit forms
- added a function to count the pending changes for any edit form
- align naming of create/edit storage pool components. they have been renamed.
- change instance/profile configuration > cloud init forms. to improve the design in them. 
- fix gpu device input. on change of gpu that is specified by id, we should not unfocus the field when typing into it
- show type of inherited devices of type "other"

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - edit instance, profile, project, network, storage pool and storage volume
    - ensure the save button shows the right number for all possible edits.
